app/oauth2.py

```diff
@@ -41,14 +44,17 @@ def verify_access_token(token: str, credentials_exception):
 
     return token_data
 
+
 # Metoda koja poziva metodu za verifikaciju tokena i vraca korisnika ciji je id jednak id koji je u payload-u tokena
-def get_current_user(token: str = Depends(oauth2_scheme), db: Session = Depends(database.get_db)):
+def get_current_user(
+    token: str = Depends(oauth2_scheme), db: Session = Depends(database.get_db)
+):
     credentials_exception = HTTPException(
         status_code=status.HTTP_401_UNAUTHORIZED,
         detail=f"Could not validate",
         headers={"WWW-Authenticate": "Bearer"},
     )
     token = verify_access_token(token, credentials_exception)
-
+    print(token)
```

Comment:
Just an info: when preparing app release, make sure tokens are not logged into the console or anywhere else